src/M6Web/Bundle/LogBridgeBundle/DependencyInjection/Configuration.php

```diff
@@ -52,6 +47,80 @@ public function getConfigTreeBuilder()
                         ->defaultValue(array())
                     ->end()
                 ->end()
+
+                ->arrayNode('filters')->isRequired()
+                    ->children()
+                        ->arrayNode('requests')
```

Comment:
The name "requests", "bad_tokens" etc. are not constant : you need to enable a dynamic configuration. The filters are differents between projetcs.

You need to use ->useAttributeAsKey and ->prototype('array') etc.